### PR TITLE
Add environment for generated graphql

### PR DESCRIPTION
**Pull request by MancunianSam**

### CODE REVIEW

File: root_github.tf [hunk 1]
```diff
@@ -85,3 +85,17 @@ module "github_checksum_repository" {
     WORKFLOW_PAT       = data.aws_ssm_parameter.workflow_pat.value
   }
 }
+
+module "github_generated_grapqhl_environment" {
```
Comment (TomJKing) [inline root_github.tf:89]: Spelling of module name, should be: `..._graphql_...`

File: root_data.tf [hunk 2]
```diff
@@ -29,3 +29,23 @@ data "aws_ssm_parameter" "workflow_pat" {
 data "aws_ssm_parameter" "slack_webhook_url" {
   name = "/mgmt/release/slack/webhook"
 }
+
+data "aws_ssm_parameter" "npm_token" {
+  name = "/mgmt/npm_token"
+}
+
+data "aws_ssm_parameter" "gpg_passphrase" {
```
Comment (TomJKing) [inline root_data.tf:37]: Is there a reason for the white space in the name of the params?

**Review by TomJKing — COMMENTED** [review]
Looks good. Couple of minor comments.

File: root_github.tf (hunk 1 above)
Comment (MancunianSam) [reply root_github.tf:89]: Oops, I've changed it.

**Review by TomJKing — APPROVED** [review]
Looks good. Thanks

**Pull request merged**